Plugin/Controller/Adminhtml/Integration/SavePlugin.php

```diff
@@ -197,7 +197,6 @@ private function sendIntegrationToExtend($integrationId, $storeId)
                     'store',
                     $storeId
                 ),
-                'magentoApiSecretId' => $this->encryptor->decrypt($oauth->getSecret()),
```

Comment:
So this is a fix because we reject additional props that aren't in the expect in the payload on the node core side?

Reply:
yeah this isn't the correct value that we need anyway, we'll get this off the integration record